.github/continuous-integration.sh

```diff
@@ -5,7 +5,7 @@ set -e # Terminates as soon as something fails
 echo "The work dir is ${ARC_MLIR_BUILD}"
 
 export PATH="$ARC_MLIR_BUILD/llvm-build/bin:$PATH"
-export RUSTC_WRAPPER="/home/arc-runner/.cargo/bin/sccache"
+export RUSTC_WRAPPER="/home/github-actions/.cargo/bin/sccache"
```

Comment:
Are you sure  /home/github-actions is persisted between invocations of the runner?

Reply:
It should be ok, this is running in a docker image on my personal computer.

Reply:
Good